src/components/ReactTags.tsx

```diff
@@ -132,6 +149,15 @@ const ReactTags = (props: ReactTagsProps) => {
     if (props.handleFilterSuggestions) {
       return props.handleFilterSuggestions(query, updatedSuggestions);
     }
+    if (enableFuzzySearch) {
+      const newSuggestionsFuzzy = fuzzySuggestions.search(query);
+
+      return props.suggestions.filter((suggestion) =>
+        newSuggestionsFuzzy.find(
+          (sug) => sug.text === suggestion.id && sug.distance < 5
```

Comment:
lets store `5` in a constant so its clear whats its purpose

Reply:
added [feat: update fuzzySearch value memo dependency](https://github.com/react-tags/react-tags/pull/985/commits/e3a80590f041b5112cab3b6938b7e12720fa4992) 

https://github.com/react-tags/react-tags/blob/e3a80590f041b5112cab3b6938b7e12720fa4992/src/components/ReactTags.tsx#L55

I had this earlier, then I thought I shouldn't have a variable 😬

Reply:
added https://github.com/react-tags/react-tags/blob/c4a741bf8512a52c265d86c24cb3282fe22fd008/src/components/ReactTags.tsx#L55